docs/src/categories.json

```diff
@@ -28,6 +28,7 @@
     "PageLayout",
     "OnThisPage",
     "BackToTopButton",
+    "DownloadButton",
```

Comment:
Nit: I wonder if DownloadButton would be better categorized as a "Features" rather than "Layout" - the Layout components typically are found within the PageLayout and/or have a more structural utility to them. What do you think?